### PR TITLE
Yourkit.JavaProfiler version 2022.3

### PR DESCRIPTION
**Pull request by U1F984**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58837)

### CODE REVIEW

File: manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3
```
Comment (Trenly) [inline manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.installer.yaml:5]:
```suggestion
PackageVersion: 2022.3.96
```
This must match what is shown in control panel. When you update this field, remember to change the name of the version folder to match

File: manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3
```
Comment (Trenly) [inline manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2022.3.96
```
This must match what is shown in control panel. When you update this field, remember to change the name of the version folder to match

File: manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3
+PackageLocale: en-US
+Publisher: YourKit GmbH
```
Comment (Trenly) [inline manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml:7]:
```suggestion
Publisher: YourKit
```
Must match Control Panel

File: manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3
+PackageLocale: en-US
+Publisher: YourKit GmbH
+PackageName: Java Profiler
```
Comment (Trenly) [inline manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml:8]:
```suggestion
PackageName: YourKit Java Profiler
```
Should match Control Panel

File: manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3
+PackageLocale: en-US
+Publisher: YourKit GmbH
+PackageName: Java Profiler
+License: All rights reserved
```
Comment (Trenly) [inline manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
LicenseUrl: https://www.yourkit.com/purchase/eula.pdf
```

File: manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3
```
Comment (Trenly) [inline manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.yaml:5]:
```suggestion
PackageVersion: 2022.3.96
```
This must match what is shown in control panel. When you update this field, remember to change the name of the version folder to match

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**